course/auth.py

```diff
@@ -680,9 +774,35 @@ def user_profile(request):
 
     user_form = None
 
+    def has_verified_inst_id(user):
```

Comment:
I think when I don't want to verify user's student's id, that method should be named ``has_inst_id``. I should have make this clean or introduce another method.